.github/workflows/ci.yaml

```
@@ -0,0 +1,135 @@
+name: Lint and Test Charts
+
+on:
+  pull_request:
+    paths:
+      - 'charts/**'
+
+jobs:
+  lint-chart:
+    runs-on: ubuntu-latest
+    outputs:
+      list-changed: ${{ steps.list-changed.outputs.changed }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      # Required by helm/chart-testing-action
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      # Required by helm/chart-testing-action
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.1.0
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed)
+          echo "$changed"
```

```suggestion
          echo "Changed? $changed"
```

---

`ct list-changed` does not resolve to a boolean

`if [[ -n "$changed" ]]` checks if the string has any length, not if it's true (https://www.man7.org/linux/man-pages/man1/test.1.html)

The output from `ct list-changed` is just the paths of the charts changes, split with newlines. So I counter-suggest this:
```suggestion
          echo "# Changed charts:"
          echo "$changed"
          echo "# End of changed charts."
```